### PR TITLE
Track remaining issues in the examples dashboard

### PR DESCRIPTION
**Pull request by thpani**
Update the examples dashboard to include links to issues and PRs wherever an item is marked with :x:

This is mostly for my own sanity, to
* ease tracking what is blocking the remaining examples to go through Apalache
* give a sense of impact that open issues have on our examples

ℹ️ I've made separate commits for each logical change; additional motivation in the commit message, wherever it felt necessary.

Before: 

Screenshot 2023-07-26 at 14 39 09

After:

Screenshot 2023-07-26 at 14 38 45

### CODE REVIEW

File: examples/.scripts/run-example.sh [hunk 1]
```diff
@@ -10,12 +10,50 @@
 
 result () {
     local cmd="$1"
-    if ($cmd &> /dev/null)
+    local args="$2"
+    local file="$3"
+
+    # Skip verification for specs that do not define a state machine
```
Comment (thpani) [inline examples/.scripts/run-example.sh:16]: @shonfeder I know you will object to the added complexity here, but all of this complexity will go away once we fix the relevant issues 😃

**Review by shonfeder — APPROVED** [review]

File: examples/.scripts/run-example.sh (hunk 1 above)
Comment (shonfeder) [reply examples/.scripts/run-example.sh:16]: If it helps you and I don't have to maintain it, who am I to object :wink:x

**Comment by shonfeder** [comment]
The dashboard is looking nice!

**Review by bugarela — APPROVED** [review]
Looks great!

**Pull request merged**